### PR TITLE
17730 scaffold power of attorney

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17730)

Now that we have a relatively firm design for the POA search app we need to scaffold out the basic structure. This PR is to add that scaffolding.

## Testing done
Unit and e2e tests will be included in a subsequent ticket

## Screenshots
Not needed yet, this is just to scaffold the app

## Acceptance criteria
- [x] The components in the ticket referenced above have been scaffolded 
- [x] A PR has been submitted for review

### CODE REVIEW

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+};
+
+export const uiSchema = {
+  'ui:description': 'Chech all that apply',
+  vso: {
+    'ui:title': 'Veteran Service Organization (VS0)',
+  },
+  representativeOfVSO: {
+    'ui:title': 'Representative in a Veteran service organization',
```
Comment (micahchiang) [inline src/applications/personalization/search-representative/config/basic-information/basicInformation.js:26]: I think Veteran service organization should be capitalized similar to vso to maintain consistency -> **V**eteran **S**ervice **O**rganization

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      type: 'boolean',
+    },
+    claimsAgent: {
+      type: 'boolean',
+    },
+  },
+};
+
+export const uiSchema = {
+  'ui:description': 'Chech all that apply',
```
Comment (micahchiang) [inline src/applications/personalization/search-representative/config/basic-information/basicInformation.js:21]: [mockup](https://xd.adobe.com/view/b80b0931-826b-413a-9b07-1efff71cae16-ed58/screen/e4f0ce18-e81f-4ab4-99aa-22fdfa96dadc) uses the title `Check at least one`

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+export const schema = {
+  type: 'object',
+  title: 'What kind of representative are you looking for?',
+  properties: {
```
Comment (micahchiang) [inline src/applications/personalization/search-representative/config/basic-information/basicInformation.js:4]: At some point, either now or later, we'll need to structure this schema in the same way we made the [686 options selection](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/cdc43792b240194fb02b270123e845bda3e74bf1/src/schemas/686c-674/schema.js#L115-L136) to ensure we can validate as a boolean group where at least one is required.

**Review by micahchiang — APPROVED** [review]

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js (hunk 3 above)
Comment (cohnjesse) [reply src/applications/personalization/search-representative/config/basic-information/basicInformation.js:4]: Yeah, I was gonna wait to see if there are any changes to the mockup before we determine the final structure. Definitely need to circle back on this later.

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js (hunk 2 above)
Comment (cohnjesse) [reply src/applications/personalization/search-representative/config/basic-information/basicInformation.js]: Yeah I took this wording from what we did on the 686c. But you are right, I updated this.

File: src/applications/personalization/search-representative/config/basic-information/basicInformation.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/personalization/search-representative/config/basic-information/basicInformation.js]: Derp, yep I updated this

File: src/applications/personalization/search-representative/config/form.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+// import fullSchema from 'vets-json-schema/dist/000-schema.json';
```
Comment (oseasmoran73) [inline src/applications/personalization/search-representative/config/form.js:1]: Is this still relevant? If not, can we remove?

File: src/applications/personalization/search-representative/config/form.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+const formConfig = {
+  rootUrl: manifest.rootUrl,
+  urlPrefix: '/',
+  submitUrl: '/v0/api',
+  trackingPrefix: 'search-representative-',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '21-22a',
+  saveInProgress: {
+    // messages: {
```
Comment (oseasmoran73) [inline src/applications/personalization/search-representative/config/form.js:24]: x2

**Review by oseasmoran73 — APPROVED** [review]

File: src/applications/personalization/search-representative/config/form.js (hunk 4 above)
Comment (cohnjesse) [reply src/applications/personalization/search-representative/config/form.js]: Removed

File: src/applications/personalization/search-representative/config/form.js (hunk 5 above)
Comment (cohnjesse) [reply src/applications/personalization/search-representative/config/form.js]: Removed

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**